Example/ReusableDemo/CollectionViewController.swift

```diff
@@ -31,40 +32,79 @@ class CollectionViewController: UICollectionViewController {
 
   // MARK: UICollectionViewDataSource
 
-  override func numberOfSectionsInCollectionView(collectionView: UICollectionView) -> Int {
-    return 3
-  }
+  #if swift(>=3.0)
+    override func numberOfSections(in collectionView: UICollectionView) -> Int {
+      return 3
+    }
+  #else
+    override func numberOfSections(in collectionView: UICollectionView) -> Int {
+      return 3
+    }
+  #endif
 
-  override func collectionView(collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
+  override func collectionView(_ collectionView: UICollectionView, numberOfItemsInSection section: Int) -> Int {
     return 5
   }
 
-  override func collectionView(collectionView: UICollectionView,
-    viewForSupplementaryElementOfKind kind: String,
-    atIndexPath indexPath: NSIndexPath) -> UICollectionReusableView {
-      let header = collectionView.dequeueReusableSupplementaryView(kind, indexPath: indexPath) as CollectionHeaderView
-      header.title = "Section \(indexPath.section)"
-      return header
-  }
+  #if swift(>=3.0)
+    override func collectionView(_ collectionView: UICollectionView,
+      viewForSupplementaryElementOfKind kind: String,
+      at indexPath: IndexPath) -> UICollectionReusableView {
+        let header = collectionView.dequeueReusableSupplementaryView(kind, indexPath: indexPath) as CollectionHeaderView
+        header.title = "Section \(indexPath.section)"
+        return header
+    }
+  #else
+    override func collectionView(_ collectionView: UICollectionView,
+      viewForSupplementaryElementOfKind kind: String,
+      at indexPath: IndexPath) -> UICollectionReusableView {
+        let header = collectionView.dequeueReusableSupplementaryView(kind, indexPath: indexPath) as CollectionHeaderView
+        header.title = "Section \(indexPath.section)"
+        return header
+    }
```

Comment:
The entire demo project has been converted to Swift 3. I figure that's what should be used when someone runs `pod try Reusable` now that Xcode 8 is out. I figure it's not worth the effort to maintain Swift 2 support for the demo project as that is mainly for devs to evaluate the library.